test/test_citusdb_baseRIwPEmodule.yml

```diff
@@ -0,0 +1,259 @@
+---
+- hosts: citus_coordinator
```

Comment:
@lel99999 We should probably set this up to run from a separate server to fully test connectivity. In Prod that would be wdcddtl01 or wdcddtl02, but we should probably have this controlled by a variable, such as hosts: {{ citus_test_client }} or something.  The CSV's must still be downloaded on the Coordinator for the COPY command to work properly without `psql`.